.changeset/purple-plants-search.md

```diff
@@ -0,0 +1,9 @@
+---
+'@primer/react-brand': patch
+---
+
+Improved default heading levels in the `FAQ` component.
+
+- Updated `FAQ.Subheading` heading level from `h3` to `h4`.
+- `FAQ.Question` now renders as a `h5` if there is a `FAQ.Subheading` present, or if rendered inside a `FAQGroup`.
```

Comment:
>`FAQ.Question` now renders as a `h5` if there is a `FAQ.Subheading` present

Doesn't seem be to be working in [this story](https://primer-875bfd87e2-26139705.drafts.github.io/brand/storybook/?path=/story/components-faq-features--with-subheadings), where the question should be a h5 not h4? Feel like this was working before, so maybe something regressed between commits?

![Screenshot 2025-06-24 at 16 43 09](https://github.com/user-attachments/assets/5c004df8-2dca-4458-9be0-8cc9354a5c4e)